_schema.graphql

```diff
@@ -1966,6 +1966,14 @@ enum CategoryType {
   INSTITUTION
 }
 
+type City {
+  slug: String
```

Comment:
we'll probably want to add more metadata here as the requirements solidify.

e.g.
`
type City {
  image: Image
}
`
also should BMW shows go in their own field? We can answer these questions later though

Reply:
Definitely — the actual nature of the BMW data is still unclear to me, but yeah a new field might make sense.